backend/src/main/java/de/qaware/mercury/rest/shop/dto/response/ShopDetailDto.java

```diff
@@ -31,6 +31,9 @@
     @Nullable
     private String imageUrl;
     @Nullable
+    private String shopColor;
```

Comment:
Wie funktioniert das?

wenn autoColorEnabled = true, dann nimm shopColor, ansonsten defaulte auf #FFFFFF im frontend? Was passiert, wenn autoColorEnabled aber shopColor == null? dann auch #FFFFFF?

Reply:
Wenn autoColorEnabled == false, ändere die Hintergrundfarbe in dem Div nicht.
Wenn autoColorEnabled == true, ändere die Hintergrundfarbe in dem Div nicht.

Same outcome.

Reply:
Der Shop hat das Feature vll. aktiv, weil das per Default an sein soll, aber es wurde noch keine Farbe berechnet, dann darf das halt nichts brechen. Per Default #FFFFFF zurückgeben will ich eigentlich nicht, weil das im Backend ne Annahme über die reguläre Hintergrundfarbe im Frontend trifft. Das gabs zwar oben bei der ImageNotFoundException gerade noch, hab ich aber inzwischen ausgebaut.

Reply:
Yo, passt. merci!